modules/packages/freebsd_ports

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh -e
+
+command=$1
+
+while read -r line; do
+  export INPUT_$line
+done
+
+list_updates() {
+  pkg version -v -l "<" | sed -e 's/\([^ ]*\)-[^-]* .* \(.*\))/Name=\1\
```

Comment:
Maybe add a comment here with an example data line and how it looks after the rewrite, for clarity.

Also this doesn't validate the `pkg` return code.  Should it?